File: easy/1/main.py
arr = [10, 15, 3, 7]

def function(value):
    # Checking if the input number can be obtained as a combination of those of the array
    for x in arr:
        if ( value - x ) in arr:
            return True
    
    return False

if function(22):
    print("True - The value can be obtained as a combination of two numbers of the array")
else:
    print("False - The value canot be obtained as a combination of two numbers of the array") 

File: easy/1/main.java
public class CodeChallenge{

    public static void main(String []args){
    
	int[] arr = {10, 15, 3, 7}; 
        
	if (function(arr, 22)) {
            System.out.println("True - The value can be obtained as a combination of two numbers of the array");
        } else {
            System.out.println("False - The value cannot be obtained as a combination of two numbers of the array");
        }
     }
     
    private static boolean function(int [] arr, int k) {
        for (int x : arr) {
            if ( arr.indexOf(k - x) > -1) {
                return true;
            }
        }
        return false;
    }
}
